README.md

```diff
@@ -7,6 +7,8 @@ Example running saucectl with XCUITest.
 The steps below illustrate one of the quickest ways to get set up. If you'd like a more in-depth guide, please check out
 our [documentation](https://docs.saucelabs.com/testrunner-toolkit/installation).
 
+_If you're using VS Code, you can try out [Runme](https://marketplace.visualstudio.com/items?itemName=stateful.runme) plugin to run following commands inside of VS Code._
```

Comment:
But that's not actually correct.

You have command statements like
```shell
APP=<your-app-ipa-file> TEST_APP=<your-test-app-ipa-file> saucectl run
```

which can't be run as is. The app and test app have to be built first as well and no command is listed for that here. So you can't actually follow along by simply running the commands in order.

It'd be good to include the xcodebuild command required here.

Reply:
👍 Let me add more details here.